templates/base/next.config.js

```diff
@@ -1,13 +1,30 @@
+require('dotenv').config();
+const withPWA = require('next-pwa');
 const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true'
 });
 
 const isAssetPrefix = process.env.BASE_PATH || '';
 
+/**
+ * This section adds the basic PWA configuration into the next config
+ * It derives from an env variable defined in .env file. Default it is enabled,
+ * To disable it you need to specify the PWA_DISABLED=true in .env file
+ * 
+ * Further PWA configuration visit https://www.npmjs.com/package/next-pwa
+ */
+const nextConfig = withPWA({
```

Comment:
For the sake of maintainability can we not maintain this PWA config under /base/config/pwa/config. ? That can export nextConfig and we can apply it here.

Reply:
I wasn't aware of , let me check